Tasks/Gradle/task.loc.json

```diff
@@ -319,14 +319,6 @@
   ],
   "instanceNameFormat": "ms-resource:loc.instanceNameFormat",
   "execution": {
```

Comment:
You are right, I was trying to piggyback on Bogdan's change as he updated the version in another PR.  However I realized if he has hotfixed the server then my changes won't be picked up.  

I should explicitly update the version myself.